app/src/main/java/org/openobservatory/ooniprobe/activity/MainActivity.java

```diff
@@ -112,4 +115,18 @@ public void onStop() {
         Countly.sharedInstance().onStop();
         super.onStop();
     }
+
+    public synchronized boolean isTestRunning() {
+        if (null == isTestRunning) {
```

Comment:
Can you invert the terms of the comparison for consistency?

Reply:
renamed in isUITestRunning